lib/compiler.js

```diff
@@ -263,6 +265,36 @@ function sortByInheritance(instructions) {
     });
 }
 
+function verifyModelDefinitions(rootDir, modelDefinitions) {
+  if (!modelDefinitions || modelDefinitions.length < 1) {
+    return undefined;
+  }
+
+  var registry = {};
+  modelDefinitions.forEach(function(definition, idx) {
+    try {
+      var fullPath = path.resolve(rootDir, definition.sourceFile);
+      definition.sourceFile = require.resolve(fullPath);
+    } catch (err) {
+      debug('Model source code not found: %s - %s', definition.sourceFile, err.code || err);
+      definition.sourceFile = undefined;
+    }
```

Comment:
This block should be run only when `definition.sourceFile` is defined. Please add a unit-test covering this case too.